.github/workflows/release.yml

```diff
@@ -139,36 +139,36 @@ jobs:
               })
```

Comment:
These code changes involve commenting out blocks of code related to extracting JIRA issue keys and fetching JIRA issue details. The original code extracted the JIRA issue key from the Pull Request description and fetched JIRA issue details using the extracted key.

The commented-out code blocks suggest that there might be a change in the workflow or the way JIRA integration is handled. The extraction of JIRA issue keys and fetching issue details from JIRA has been disabled in the updated version.

It seems that the script is now focusing on getting release information, updating release notes, and handling commit details instead of interacting with JIRA directly.

If JIRA integration is still required, the commented-out code blocks can be uncommented to re-enable the extraction and fetching of JIRA issue details. Otherwise, the script has been modified to exclude this functionality.